examples/tree-view/src/Node/Node.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
```

Comment:
:hand: Let's rename the folder itself from `Node` to `nodes`.  `Node.js` is fine.